src/controllers/trip.controller.js

```diff
@@ -221,5 +222,82 @@ class TripController {
     if (data) return response.successMessage(res, 'success', 200, data);
     return response.errorMessage(res, 'Trip not found', 404);
   }
+
+  /**
+   * This controller helps to know the type of trip
+   * sothat it can redirect the request to a tageted controller
+   * @param {Object} req request
+   * @param {*} res response
+   * @returns {Object} return user response
+   */
+  static async combineUpdateTripRequestConctroller(req, res) {
```

Comment:
what do you mean by combineUpdateTripRequestController i would like if you kindly check the name twice and rename the function according to what it is doing. and since it is a function inside class called TripController i think there no need of calling it controller.